django/viewsets/README.md

```diff
@@ -280,7 +280,7 @@ Use HttpResponseStatus instead of status number
 from rest_framework import status
 class UserViewSet(views.APIView):
     def get(self, request):
-        return Response(data={}, status=status.HTTP_200_OK)
+        return Response(data={}) # Defaults to 200
```

Comment:
Maybe more explanation na? Like default status is 200 or something.

Reply:
What's the point krub?  This example is show how to proper use the status code.

Reply:
Oh, so maybe we might need to show another status as an example, and tell that there is no need to put 200 there since it is a default krub?

Reply:
I will try.